packages/cozy-client/src/store/documents.js

```diff
@@ -90,15 +90,17 @@ export const getDocumentFromSlice = (state = {}, doctype, id) => {
   if (!state[doctype]) {
     if (process.env.NODE_ENV !== 'production') {
       console.warn(
-        `getDocumentFromSlice: ${doctype} is absent from the store documents`
+        `getDocumentFromSlice: ${doctype} is absent from the store's documents. State is`
       )
+      console.warn(state)
```

Comment:
Nit: if you pass multiple arguments to `console.warn`, they get printed each after the other. So `console.warn(message, state)` would work too :)

Reply:
Done